lib/smart_answer/calculators/arrested_abroad.rb

```diff
@@ -1,14 +1,12 @@
 module SmartAnswer::Calculators
   class ArrestedAbroad
     # created for the help-if-you-are-arrested-abroad calculator
-    attr_reader :data
+    attr_accessor :country
```

Comment:
`country` should be set of initialization. It is fundamental to the way the class works. So:
```ruby
attr_reader :country

def initialize(country)
  @country = country
end
```